cfgov/cfgov/urls.py

```diff
@@ -41,6 +41,7 @@
 
 fin_ed = SheerSite('fin-ed-resources')
 oah = SheerSite('owning-a-home')
+es_conv_flag = partial(flagged_url, 'ES_CONV_FLAG')
```

Comment:
This is a really neat way to do this! I like it a lot. Might have to add it as a possible pattern to the documentation.